tests/compute/groupshared.slang

```diff
@@ -1,39 +1,32 @@
 // groupshared.slang
-
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute
 //TEST(compute):COMPARE_COMPUTE_EX:-slang -compute -dx12
 //TEST(compute, vulkan):COMPARE_COMPUTE_EX:-vk -compute
 //TEST(compute):COMPARE_COMPUTE_EX:-cuda -compute
-
```

Comment:
There are some spurious whitespace diffs in this change (the removed lines here, and then indentation changes later on), possibly as a result of how the modified test file got shared. It might be nice if the only diffs in the test where the ones to set up `THREAD_COUNT` as a `static const` variable instead of a `#define`.